src/main/java/io/vertx/core/http/impl/Http1xClientConnection.java

```diff
@@ -87,6 +87,7 @@
   private int keepAliveTimeout;
   private int seq = 1;
   private long bytesRead;
+  private long initialTimestamp;
```

Comment:
make it final

Reply:
@vietj if I will make it final then will have to initialise it.
Then will not be able to do 
```
if (options.isKeepAliveTTLEnabled())
      initialTimestamp = System.currentTimeMillis();
```

Reply:
you can init it with `0L`

Reply:
Yes but then cannot reassign it in constructor when options.isKeepAliveTTLEnabled() is true as 
```
initialTimestamp = System.currentTimeMillis();
```